python/delta/tests/test_deltatable.py

```diff
@@ -966,6 +980,16 @@ def test_delta_table_builder_with_bad_args(self) -> None:
         with self.assertRaises(TypeError):
             builder.partitionedBy([1])  # type: ignore[list-item]
 
+        # bad clusterBy col name
```

Comment:
Let's put these into a function, maybe `test_cluster_by_bad_args`

Reply:
This test tests every other builder function, seems weird to make a dedicated test just for clusterBy args